### PR TITLE
save _horizon during setup

### PR DESCRIPTION
**Pull request by vmoens**
cc @vikashplus

### CODE REVIEW

File: robohive/envs/env_base.py [hunk 1]
```diff
@@ -136,6 +136,9 @@ def _setup(self,
         assert not done, "Check initialization. Simulation starts in a done state."
         self.observation_space = gym.spaces.Box(obs_range[0]*np.ones(observation.size), obs_range[1]*np.ones(observation.size), dtype=np.float32)
 
+        # we copy the horizon to avoid having gymnasium hijacking it later
+        self._horizon = self.spec.max_episode_steps
```
Comment (vikashplus) [inline robohive/envs/env_base.py:140]: `self.spec` is none at this point.

**Comment by vmoens** [comment]
@vikashplus hopefully this should work, or at least a version of that with `implement_for` if `gym.registry` syntax changes from release to release

**Comment by vmoens** [comment]
The checks are passing but better check on your end if it's all ok :)

**Review by vikashplus — APPROVED** [review]
Thanks for helping with this. 

LGTM

**Pull request merged**